test/functional/fixtures/api/es-next/select/pages/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Select</title>
+</head>
+<body>
+<input id="input" value="input"/>
+
+<div id="div" contenteditable="true">div<p id="p1">with</p>contenteditable<p id="p2">attribute</p></div>
+
+<textarea id="textarea">
+123
+456
+</textarea>
+
+<div id="status"></div>
+
+<script type="text/javascript">
+    document.getElementById('div').addEventListener('mouseup', function () {
+        var selection = window.getSelection();
+        var startNode = this.childNodes[1].childNodes[0];
+        var endNode   = this.childNodes[3].childNodes[0];
+
+        if (selection.anchorNode === startNode && selection.anchorOffset === 0 && selection.focusNode === endNode &&
+            selection.focusOffset === endNode.nodeValue.length)
+            document.getElementById('status').textContent = 'Selected editable content';
+    });
+
+    document.getElementById('input').addEventListener('mouseup', function () {
+        if (this.selectionStart === 2 && this.selectionEnd === 4)
+            document.getElementById('status').textContent = 'Selected text in input';
+    });
+
+    document.getElementById('textarea').addEventListener('mouseup', function () {
+        if (this.selectionStart === 2 && this.selectionEnd === 7)
```

Comment:
Maybe we will just use hybrids which will return `selectionStart` and `selectionEnd` and assert their values instead of status?